packages/shopify-app-remix/src/server/authenticate/admin/__tests__/doc-request-path.test.ts

```diff
@@ -43,138 +43,6 @@ describe('authorize.admin doc request path', () => {
       expectLoginRedirect(response);
     });
 
-    it('redirects to auth when not embedded and there is no offline session', async () => {
```

Comment:
These tests have been moved to either `manage-access-token.test.ts` or `ensure-installed-on-shop.test.ts`